pages/viewPlans/index.js

```diff
@@ -0,0 +1,7 @@
+export default function ViewAllPlans() {
+    return (
+        <>
+        <div>Here Will Be All Plans</div>
+        </>
+    )
```

Comment:
you don't need fragments:

```suggestion
    return <div>Here Will Be All Plans</div>
```